assemblyline/common/tag_whitelist.yml

```diff
@@ -39,7 +51,15 @@ regex:
   network.dynamic.uri:
     - (?:ftp|http)s?://localhost(?:$|/.*)
     - (?:ftp|http)s?://(?:(?:(?:10|127)(?:\.(?:[2](?:[0-5][0-5]|[01234][6-9])|[1][0-9][0-9]|[1-9][0-9]|[0-9])){3})|(?:172\.(?:1[6-9]|2[0-9]|3[0-1])(?:\.(?:2[0-4][0-9]|25[0-5]|[1][0-9][0-9]|[1-9][0-9]|[0-9])){2}|(?:192\.168(?:\.(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]|[0-9])){2})))(?:$|/.*)
+    - (?:http)?:\/\/schemas\.(microsoft|openxmlformats)\.(com|org)(?:$|\/.*)
+    - (?:http)?:\/\/schemas\.android\.com\/apk\/res(-auto|\/android)
+    - (?:http)?:\/\/xmlpull\.org\/v1\/doc\/features\.html(?:$|.*)
+    - (?:http)s?:\/\/android\.googlesource\.com\/toolchain\/llvm-project
   # Regular expression to whitelist static URIs (Private IPs and localhost)
   network.static.uri:
     - (?:ftp|http)s?://localhost(?:$|/.*)
     - (?:ftp|http)s?://(?:(?:(?:10|127)(?:\.(?:[2](?:[0-5][0-5]|[01234][6-9])|[1][0-9][0-9]|[1-9][0-9]|[0-9])){3})|(?:172\.(?:1[6-9]|2[0-9]|3[0-1])(?:\.(?:2[0-4][0-9]|25[0-5]|[1][0-9][0-9]|[1-9][0-9]|[0-9])){2}|(?:192\.168(?:\.(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]|[0-9])){2})))(?:$|/.*)
+    - (?:http)?:\/\/schemas\.(microsoft|openxmlformats)\.(com|org)(?:$|\/.*)
+    - (?:http)?:\/\/schemas\.android\.com\/apk\/res(-auto|\/android)
```

Comment:
You don't need ? after (?:http) because that makes http optional which in this case it's not

Reply:
Addressed in 10bc286fa6d819aaef9c190e57a5cecf71294139